packages/ifpack/src/euclid/guards_dh.h

```diff
@@ -44,15 +44,15 @@
 #define GUARDS_DH
 
 #include "euclid_common.h"
-
+#include "Ifpack_GlobalComm.h"
 
 /*
    This file defines the INITIALIZE_DH and FINALIZE_DH macros
 */
 
 #define  INITIALIZE_DH(argc, argv, help) \
             MPI_Init(&argc,&argv);  \
-            comm_dh = MPI_COMM_WORLD;    \
+            comm_dh = ifpack_get_global_comm();    \
```

Comment:
Since this macro initializes MPI, I don't think using `MPI_COMM_WORLD` is incorrect here